nservicebus/upgrades/5to6-async-suffix.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Async Suffix
+summary: Reasoning for no Async suffix to Task based APIs
+reviewed: 2016-07-21
+related:
+ - nservicebus/upgrades/5to6
+---
+
+
+## No Async Suffix for NServiceBus APIs
+
+Starting with NServiceBus Version 6, all APIs that contain potentially IO blocking code are [Async](https://msdn.microsoft.com/en-us/library/mt674882.aspx). Some examples include:
+
+ * Endpoint messaging methods such as [Send and Publish](/nservicebus/upgrades/5to6.md#message-handlers-bus-send-and-receive)
+ * [Sagas and Message Handlers](/nservicebus/upgrades/5to6.md#message-handlers)
+ * [Message pipeline extension points](/nservicebus/upgrades/5to6.md#pipeline-customization).
+ * [Endpoint Start and Stop](/nservicebus/upgrades/5to6.md#endpoint-start-and-stop).
+ * [Message mutators](/nservicebus/upgrades/5to6.md#pipeline-customization-message-mutators).
+
+Note that none of these APIs have the *Async* suffix as recommended by the Microsoft convention, which states: 
```

Comment:
When I first read I had troubles to see the "none" in the sentence because we start with Note that none... Could we rephrase to start without a negation?